students/shayna_andersonhill/lesson02/grid_printer.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python3
+#Shayna Anderson-Hill
+#Script to write a function that draws a grid
+#01-22-2018
+
+plus = '+'
+minus = '-'
+space = ' '
+line = '|'
+
+#Naive Answer
+print('Part One')
+for i in range(1,3):
+    print(plus, (minus+space)*4, plus, (minus+space)*4, plus)
+    for i in range(1,5):
+        print(line, space*8, line, space*8, line)
+print(plus, (minus+space)*4, plus, (minus+space)*4, plus)
+
+#Create function to print grid customized by input value n
+
+def horizontals(n, columns=2):
+    print(plus, (minus+space)*(n-1)+minus, plus, (minus+space)*(n-1)+minus, plus)
+
+def verticals(n):
+    for i in range(n):
+        print(line, space*((n*2)-1), line, space*((n*2)-1), line)
+
+print('Part 2:')
+def print_standard_grid(n, size = 2):
+    for i in range(1,3):
+        horizontals(n)
```

Comment:
So in this instance, you didn't need to call a function to execute a single line.  better to just keep the original line even if you copy paste it to use it elsewhere.  We use functions primaries for multiple statements (with exceptions of course).